crates/cervo-runtime/Cargo.toml

```diff
@@ -17,6 +17,11 @@ readme = "../../README.md"
 cervo-core = { version = "0.4.1-alpha.0", path = "../cervo-core"}
 thiserror = "1.0"
 anyhow = "1.0.41"
+rayon = "1.5"
 
 [dev-dependencies]
 cervo-asset = { version = "0.4.1-alpha.0", path = "../cervo-asset"}
+
+[features]
+threaded = []
+default = ["threaded"]
```

Comment:
Default to off - much better to opt-in when you want it. Also, I'd make rayon optional and only active if threaded is on.